tests/saw/spec/s2n_handshake_io.cry

```diff
@@ -55,7 +55,9 @@ advance_message conn = conn2
         (handshake2 : handshake) = {handshake_type = conn.handshake.handshake_type
                                    ,message_number = conn.handshake.message_number + 1
                                    }
-        cork2 = if ((ACTIVE_STATE conn2).writer != (PREVIOUS_STATE conn2).writer)
+        cork2 = if ((ACTIVE_STATE conn2).writer != (PREVIOUS_STATE conn2).writer
+                && (ACTIVE_STATE conn2).writer != 'A' && (PREVIOUS_STATE
+                    conn2).writer != 'A')
```

Comment:
nit: this line break is a bit hard to read - consider moving `&& (PREVIOUS_STATE` on a next line as well

Reply:
All of the SAW related files have been fixed by Galois folks and can be found in https://github.com/songhu/s2n/pull/5.

Hmmm, how come I can't replay to the above comments. Anyway, I am responding here:
'How about "application input"?'
Changed to ....APPLICATION_INPUT, and application input